gcp/container/client.go

```diff
@@ -12,7 +12,7 @@ type client struct {
 	logger  logger
 
 	service    *gcpcontainer.Service
-	containers *gcpcontainer.ProjectsZonesClustersService
+	containers *gcpcontainer.ProjectsLocationsClustersService
```

Comment:
Likely as a result of having had to split the main branch up into separate branches (sorry!), this one is missing the go.mod/go.sum changes to bring in the newer version of the google.golang.org/api

Reply:
Its my bad, i thought i actually caught that specific case but forgot to commit them here, sorry! I pushed them up

Reply:
@Alphasite Sorry, hopefully last request - there are conflicts with the go mod/sum as of some recent changes if you wouldn't mind rebasing off main and resolving?

Reply:
Done!